packages/block-editor/src/components/block-switcher/style.scss

```diff
@@ -3,15 +3,15 @@
 	height: $icon-button-size;
 }
 
-.components-icon-button.block-editor-block-switcher__toggle,
-.components-icon-button.block-editor-block-switcher__no-switcher-icon {
+.components-button.block-editor-block-switcher__toggle,
+.components-button.block-editor-block-switcher__no-switcher-icon {
```

Comment:
Should I expect to see the `.has-icon` modifier class here?

I see there's lots of examples where we're not including the modifier class when removing the `icon-` portion. Is that intentional?

Reply:
I wanted to avoid the extra specificity when not needed. In most of these examples the extra has-icon has no impact we just want more specificity over the button styles;